frontend/controllers/TasksController.php

```diff
@@ -122,46 +123,83 @@ public function actionView($id = null)
 			'detail' => $detail,
 			'count_tasks' => $count_tasks,
 			'user' => $user_created_at,
-            'actionResponseForm' => $actionResponseForm,
-            'actionRefuseForm' => $actionRefuseForm,
-            'actionDoneForm' => $actionDoneForm
+			'actionResponseForm' => $actionResponseForm,
+			'actionRefuseForm' => $actionRefuseForm,
+			'actionDoneForm' => $actionDoneForm,
+			'resp' => $resp
 		]);
 	}
 
-	public function actionCreate() {
-        // По умолчанию, после регистрации пользователю присваивается роль «Заказчик». Чтобы стать исполнителем необходимо
-        // отметить хотя бы одну специализацию у себя в профиле.
-        // Соответственно, при отмене всех галочек пользователь вновь становится исполнителем.
-	    $speciality = UserCategory::find()->where(['user_id' => Yii::$app->user->id])->column();
-        //проверяем, является ли пользователь заказчиком
 
-        if(count($speciality) > 0) {
-            return $this->render(Yii::getAlias('@web') . '/site/error', [
-                'name' => 'Доступ к созданию задачи запрещен',
-                'message' => 'Вы являетесь исполнителем. Пожалуйста, уберите специализацию в настройках профиля'
-            ]);
-        }
+	public function actionRefuse($id = null)
+	{
+
+
+		if (Yii::$app->request->getIsPost()) {
+			$user_id = Yii::$app->request->post('user_refused');
+			$response = Response::findOne(['task_id' => $id, 'user_id' => $user_id]);
+
+			if (isset($response)) {
```

Comment:
Тут надо поменять наоборот. Если его нет, тогда ошибка
```suggestion
			if (null === $response) {
```